src/portal/src/app/shared/components/navigator/navigator.component.html

```
@@ -111,7 +111,16 @@
             href="javascript:void(0)"
             id="aboutMenu"
-            class="nav-link nav-text nav-about-link"
+            class="nav-link nav-text"
+            (click)="logIn()"
+            *ngIf="!isSessionValid"
+            >{{ 'ACCOUNT_SETTINGS.LOGIN' | translate }}</a
+        >
+        <div class="nav-divider"></div>
+        <a
+            href="javascript:void(0)"
+            id="aboutMenu"
```

HTMLHint (1.9.2)

[error] The id value [ aboutMenu ] must be unique.

(id-unique)